src/js/components/Institutions.jsx

```diff
@@ -109,26 +155,47 @@ export default class Institution extends Component {
         pathname={this.props.location.pathname}
         userName={this.props.user.profile.name} />
       <div id="main-content" className="usa-grid">
-        <UserHeading period="2017" userName={this.props.user.profile.name} />
+        <UserHeading
+          period="2017"
+          userName={this.props.user.profile.name} />
         <div className="usa-width-two-thirds">
           {this.props.filings.map((filingObj, i) => {
             const filing = filingObj.filing
+            const latestSubmissionStatus = filingObj.submissions[0].status || null
```

Comment:
Following up on this, I filed https://github.com/cfpb/hmda-platform/issues/819

@schbetsy said that under most circumstances, the order will be what we'd expect here, so I'll sign off on this (after sneaking in a guard)